include/gz/utils/detail/subprocess.h

```diff
@@ -923,6 +923,17 @@ FILE *subprocess_stderr(const struct subprocess_s *const process) {
   }
 }
 
+int subprocess_signal(const subprocess_s *const process,
+                      int signum)
+{
+#if defined(_WIN32)
+  return GenerateConsoleCtrlEvent(signum, process->hProcess);
```

Comment:
Seems like this is not compiling on windows.

Reply:
Yeah, I haven't had a chance to boot my laptop today.  will take a look in a bit.

Reply:
This should be resolved now.

Reply:
Actually, hold, this doesn't fix the test I was trying to fix.